static/app/components/u2f/u2finterface.tsx

```diff
@@ -111,18 +130,53 @@ class U2fInterface extends React.Component<Props, State> {
       });
   }
 
+  webAuthnSignIn(authenticateRequests) {
+    const credentials = [] as PublicKeyCredentialDescriptor[];
```

Comment:
You actually probably want to write this as 

```suggestion
    const credentials: PublicKeyCredentialDescriptor[] = [];
```